### ISSUE TITLE
Go to video fullscreen when device is in landscape mode

### ISSUE THREAD

**Issue by belst** [w0bm/w0bm.com#262]
tilting your phone to landscape mode should make the video fullscreen

**Comment by jkhsjdhjs** [comment]
yeah whatever, just assign me to every issue xD

edit: but shouldn't be hard to implement

**Comment by Flummi** [comment]
jkhsjdhjs will fix it xDDDDD

**Comment by sirx** [comment]
xDDDDDDDDDDDDDDDDDD

![flummer](https://user-images.githubusercontent.com/14188379/37331581-927b6460-26a4-11e8-9d2c-b234113a8b49.png)

**Comment by jkhsjdhjs** [comment]
With following code this feature would work in chrome:
```javascript
if($.browser.mobile) {
    document.addEventListener("fullscreenerror", function(e) {
        alert(JSON.stringify(e));
    });
    window.screen.orientation.onchange = function() {
        if(this.type.startsWith("landscape"))
            player.requestFullscreen();
        else
            player.exitFullscreen();
    };
}
```

However, it wouldn't work in firefox. You'd have to enable `full-screen-api.unprefix.enabled` in `about:config` first, and then the `fullscreenerror` event would fire, which means that firefox denies us the fullscreen mode. Any ideas on how to fix this?

**Comment by belst** [comment]
it's a security feature of firefox :unamused:

**Comment by jkhsjdhjs** [comment]
Will also be allowed in firefox soon: https://bugzilla.mozilla.org/show_bug.cgi?id=1257029

**Comment by GottZ** [comment]
just toggle it anyways and see if you are in fullscreen after the request.  
if not -> throw an error so we know about the browser versions that don't support it in sentry.

**Comment by jkhsjdhjs** [comment]
Implemented with #264
